Makefile

```diff
@@ -17,4 +17,4 @@ deps: submodules
 	cd era-compiler-tester && zksync-llvm clone && zksync-llvm build
 
 test:
```

Comment:
Running from a clean `clone` (`git clean -xdf`):
```bash
➜  era_vm git:(fix-deployer) ✗ make test
export LLVM_SYS_170_PREFIX=/Users/jm-lambda/Coding/zk/era_vm/era-compiler-tester/target-llvm/target-final/ && cd era-compiler-tester && cargo run --verbose --features lambda_vm --release --bin compiler-tester -- --path  tests/solidity/simple/yul_instructions/ --target EraVM --mode='Y+M3B3 0.8.26'
/bin/sh: line 0: cd: era-compiler-tester: No such file or directory
make: *** [test] Error 1
```

We should add `deps` as a `test` make dependency:
```Makefile
test: deps
```